### PR TITLE
perf: slightly faster key up/down

### PR DESCRIPTION
**Pull request by kulmann**
## Description
Keyboard interaction with file listings needs a major rework (should be a composable instead of a component + is not compatible with the tiles view), so I didn't dig deeper. This PR is just a slight improvement of the render performance of the current implementation of up/down key uses, since it was a low hanging fruit.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-files/tests/unit/components/FilesList/KeyboardActions.spec.ts [hunk 1]
```diff
@@ -70,14 +70,14 @@ describe('KeyboardActions', () => {
...
-        '</table>'
-    })
+    // beforeEach(() => {
+    //   document.body.innerHTML =
+    //     '<table>' +
+    //     '<tr class="oc-tbody-tr" data-item-id="0"><td></td></tr>' +
+    //     '<tr class="oc-tbody-tr" data-item-id="1"><td></td></tr>' +
+    //     '<tr class="oc-tbody-tr" data-item-id="2"><td></td></tr>' +
+    //     '</table>'
+    // })
```
Comment (JammingBen) [inline packages/web-app-files/tests/unit/components/FilesList/KeyboardActions.spec.ts:80]:
```suggestion
```

**Review by JammingBen — CHANGES_REQUESTED** [review]
I found an edge case:

1. Select files via "shift" + "arrow down" until you reach the end of the files list.
2. Now try to navigate up again without shift -> will work one time, then it's broken.

**Review by JammingBen — APPROVED** [review]
